github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -744,3 +744,61 @@ orgs:
         repos:
           cloud-image-builder: admin
           community: admin
+  openshift-cnv:
+    admins:
+    - dhiller
+    - fabiand
+    - fgimenez
+    - kubevirt-bot
+    - tiraboschi
+    billing_email: fdeutsch@redhat.com
+    company: ""
+    default_repository_permission: read
+    description: OpenShift CNV
+    email: ""
+    has_organization_projects: true
+    has_repository_projects: true
+    location: ""
+    members:
+    - kubevirt-commenter-bot
+    - openshift-ci-robot
+    - openshift-merge-robot
+    - orenc1
+    - stu-gott
+    - zcahana
+    members_can_create_repositories: false
+    name: OpenShift CNV
+    repos:
+      cnv-ci:
+        allow_merge_commit: false
+        allow_rebase_merge: false
+        description: OpenShift CNV CI - files and scripts to support testing CNV on OpenShift CI
+        has_projects: true
+    teams:
+      cnv-ci:
+        description: ""
+        maintainers:
+        - dhiller
+        - fabiand
+        members:
```

Comment:
Do we need `kubevirt-bot` as member?

Reply:
Don't think so as kubevirt-bot is already an admin: https://github.com/kubevirt/project-infra/blob/d4eb1dffc82113f9c517d715aa448bda80262fdc/github/ci/prow-deploy/files/orgs.yaml#L752

WDYT?

Reply:
But I think this points towards another problem: we should streamline our configuration for KubeVirt org also

Reply:
Note that kubevirt-bit is only in one team (maintainers) and in none other: https://github.com/kubevirt/project-infra/blob/d4eb1dffc82113f9c517d715aa448bda80262fdc/github/ci/prow-deploy/files/orgs.yaml#L752

Reply:
> Don't think so as kubevirt-bot is already an admin:

yep, makes sense :+1: 

> But I think this points towards another problem: we should streamline our configuration for KubeVirt org also

Agree, we should align the configs

Reply:
I've done another reorg for this org: I've added two bot teams so that we can add access to any repo more easily (not really required yet, but interesting as a pattern for KubeVirt org maybe)

WDYT?

Reply:
I think @hco-bot should also be included here, with permissions to commit directly to test every new build with postsubmit jobs.